<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>459</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>459</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="tooth_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="clear_pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>delete tooth data</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/eraser.png</normaloff>:/eraser.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QFrame" name="editframe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QComboBox" name="comments_comboBox">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KUO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mobile Tooth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Early Caries</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Filling Missing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chipped</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cracked</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OTHER</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0" rowspan="5">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>130</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="am_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>AM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="co_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>CO</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="gl_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>GL</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="porc_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Po</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="gold_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="leftTooth_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Apply and move Back a tooth</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/back.png</normaloff>:/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Apply &amp; Add Another Item</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="rightTooth_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Apply &amp; Move to Next Tooth</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/forward.png</normaloff>:/forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="static_pushButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Toggle between static and plan</string>
     </property>
     <property name="text">
      <string>Static</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="dec_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>toggle between deciduous and permanent tooth</string>
     </property>
     <property name="text">
      <string>Dec</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="at_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>denture tooth</string>
     </property>
     <property name="text">
      <string>AT</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="tm_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>tooth missing</string>
     </property>
     <property name="text">
      <string>TM</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="plan_pushButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Toggle between static and plan</string>
     </property>
     <property name="text">
      <string>Plan</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="ex_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>extract (plan only!)</string>
     </property>
     <property name="text">
      <string>Ex</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="rt_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>root treatment</string>
     </property>
     <property name="text">
      <string>RT</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="defaultCrown_pushButton">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>porcelain veneer</string>
     </property>
     <property name="text">
      <string>Cr</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="comp_pushButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Toggle between static and plan</string>
     </property>
     <property name="text">
      <string>Comp</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QComboBox" name="fs_comboBox">
     <item>
      <property name="text">
       <string>--Fissure Sealants</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fissure Sealant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fissure Sealant/Composite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QComboBox" name="br_comboBox">
     <item>
      <property name="text">
       <string>--Bridge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bonded Retainer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bonded Pontic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adhesive Wing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lava Retainer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lava Pontic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gold Retainer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gold Pontic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other Retainer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other Pontic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QComboBox" name="endo_comboBox">
     <item>
      <property name="text">
       <string>--Endo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pulp Extirpation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Root Canal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QComboBox" name="ex_comboBox">
     <item>
      <property name="text">
       <string>--Surgical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extraction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Surgical Extraction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Apicectomy</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QComboBox" name="static_comboBox">
     <item>
      <property name="text">
       <string>--Static items</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tooth Missing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Denture Tooth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Root Present</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
